### ISSUE TITLE
Feature request: Browser support for WebSocketClient API

### ISSUE THREAD

**Issue by ukch** [theturtle32/WebSocket-Node#309]
At present, AFAICT (the docs are a bit vague on the issue), the browser does not support the `WebSocketClient` API, just the `W3WebSocket` API. One thing that would be great would be to be able to convert my existing Node.JS into web applications without needing to rewrite all of the websocket code. Would it be possible to write a `WebSocketClient` implementation that is a wrapper around `W3WebSocket`?

**Comment by ibc** [comment]
I don't think that makes many sense. I understand that you already have WebSocket code based on `WebSocketClient` API in your Node.js, but the browser exposes a standard W3C API and every browser-side app uses that API and not a wrapper of yet another WebSocket API.
